### ISSUE TITLE
Option to not generate nested keys when declaring a string with a dot

### ISSUE THREAD

**Issue by vricosti** [pixari/vue-i18n-extract#130]
Hi,

When I declare the following code to translate:

login.vue
```
<div class="login-wrap-title">
    <label>{{ $t('login.title') }}</label>
</div>
```

And I call  yarn run vue-i18n-extract use-config I get the following output:

```
"login": {
    "title": ""
  },

```
Would it be possible to add an option to generate keys that are not nested ?
Something like:

"login.title": ""

Maybe my request is coimpletely stupid (I come from a c++ background) but for some reason I would like to identify where is located the key.
Of course an alternative is to use an underscore instead of a dot (login_title) but I found the dot to be more elegant.

**Comment by DennisBetawerk** [comment]
This would be required even to support the use case where Vuei18n is configured with formatFallbackMessages: true. We use this so that we can use the default language strings themselves as the keys.

**Comment by vricosti** [comment]
Is this project still maintained ? 
Could someone please merge this PR https://github.com/pixari/vue-i18n-extract/pull/106.
Should not be difficult to fix the merges issues.
I really this option.
Thanks

**Comment by pixari** [comment]
Hi @vricosti!
Thank you for your support.
v2.0 has been merged! 🥳 

As soon as the conflicts will be fixed, we are going to merge the #133. :)

Thank you again!

**Comment by Spittal** [comment]
As of 2.0.1 you can set `--separator` as en empty string and that will achieve the functionality you want!
